src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .split(';')
+    .filter((pair) => pair.trim() !== '');
+
+  const stylesObject = {};
+
+  stylePairs.forEach((pair) => {
+    const [key, value] = pair.split(':').map((part) => part.trim());
+
+    stylesObject[key] = value;
+  });
+
+  return stylesObject;
```

Comment:
Returning the `stylesObject` is correct and follows the expected function behavior.